src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4139,6 +4142,8 @@ def remove_hc(cmd, resource_group_name, name, namespace, hybrid_connection, slot
                                                                   hybrid_connection, slot)
     return return_hc
 
+def list_functionapp_vnet_integration(cmd, name, resource_group_name, slot=None):
```

Comment:
Do we really need to create `list_functionapp_vnet_integration` and `remove_functionapp_vnet_integration`? It looks to me like we could just use `list_vnet_integration` and `remove_vnet_integration`

Reply:
The reason I want to use `list_functionapp_vnet_integration` is that I want to separate the logics for functionapp and webapp, as functionapp will have more independent logic in the future to support centauri and I don't want to mess up with webapp logic while we are adding new feature. It will also be clear for future developer to use separate method for functionapp and webapp. But if you feel we can combine them, I can change it. LMK what you think.

Reply:
Got cha. Makes sense. Sounds good to me.